CMakeLists.txt

```diff
@@ -756,3 +766,30 @@ add_subdirectory(build_tools/benchmarks)
 if(IREE_BUILD_SAMPLES)
   add_subdirectory(samples)
 endif()
+
+if(IREE_BUILD_TESTS)
+  get_property(IREE_TEST_TMPDIRS_REQUIRED GLOBAL PROPERTY IREE_TEST_TMPDIRS_REQUIRED)
+  set(IREE_CREATE_TEST_TMPDIRS_COMMANDS "")
+  set(_CMD_PREFIX "\"cmake -E make_directory")
+  set(_CUR_CMD "${_CMD_PREFIX}")
+  set(_CMD_LEN_LIMIT 8191)
+  foreach(_DIR IN LISTS IREE_TEST_TMPDIRS_REQUIRED)
```

Comment:
Could pull this code out into `iree_macros.cmake` (to keep the root file a _little_ smaller :P)

Reply:
Yeah will do